core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -71,7 +70,7 @@ class CloudSqlInstance {
   private boolean forceRefreshRunning;
 
   static final RateLimiter defaultRateLimiter() {
-    return RateLimiter.burstyBuilder(2, Duration.ofSeconds(30)).build();
+    return RateLimiter.create(2.0 / 60.0); // 2 requests/minute
```

Comment:
We'll 1 request every 30 seconds to mostly match the other connectors.

Reply:
1 request every 30 seconds is exactly what this means.

Reply:
Why not write 30.0 then? Also the comment is misleading.

Reply:
The original PR from long ago had 2.0/60.0.  I have updated this PR for clarity to be 1.0/30.0.